### ISSUE TITLE
Topic 4 Color Contrast issue

### ISSUE THREAD

**Issue by vprentice** [Section508Coordinators/DHSA-TT-200-A#49]
## Description
It was noted during testing of (Topic 4) that he  ANDi tool has identified that there is text on the page that does not meet color contrast ratio requirement of 4.51. 

## Location
https://training.section508testing.net/mod/book/view.php?id=1229&chapterid=2676

**Comment by drewnielson** [comment]
Fixed

**Comment by vprentice** [comment]
Still seeing the issue of Color Contrast:

https://training.section508testing.net/mod/book/view.php?id=1238&chapterid=2824

**Comment by katbecker** [comment]
Sorry. I made an assumption that the styles cascaded down from the default template, but they don't. Should be fixed on sub pages now, too.

**Comment by vprentice** [comment]
Thanks!
